docs/source/tutorials/index.rst

```diff
@@ -6,9 +6,8 @@ Tutorials
 
    datalab/
    indepth_overview
+   clean_learning/
```

Comment:
what are your thoughts about having this above indepth_overview actually?

CC @elisno 

That way the datalab/ and clean_learning/ pull downs contain our primary recommended quickstart tutorials for this package. 

I'm not sure we want somebody's early experience to be indepth_overview tutorial (maybe intimidating).

Reply:
I think that would look cleaner personally. And the organization order makes sense. Will defer to @elisno if he agrees.